inc/Engine/Media/Lazyload/CSS/Front/FileResolver.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace WP_Rocket\Engine\Media\Lazyload\CSS\Front;
+
+class FileResolver {
+
+	/**
+	 * Resolves the name from the file from its URL.
+	 *
+	 * @param string $url URL from the file to resolve.
+	 * @return string
+	 */
+	public function resolve( string $url ): string {
```

Comment:
Is this method goal to get the absolute path of a file based on the URL? If yes, we have a function doing that already `rocket_url_to_path( $url )`